src/ProjectManager.cpp

```diff
@@ -249,14 +249,13 @@ void InitProjectWindow( ProjectWindow &window )
    // irrespective of the order in which they were created.
    ToolManager::Get(project).GetTopDock()->MoveBeforeInTabOrder(&ruler);
 
-   const auto pPage = window.GetMainPage();
 
    wxBoxSizer *bs;
    {
       auto ubs = std::make_unique<wxBoxSizer>(wxVERTICAL);
       bs = ubs.get();
       bs->Add(topPanel, 0, wxEXPAND | wxALIGN_TOP);
-      bs->Add(pPage, 1, wxEXPAND);
+      bs->Add(window.GetContainerWindow(), 1, wxEXPAND);
```

Comment:
Notice that the bottom toolbar docking area is part of the right side of the container window, therefore it will be narrower when the effects panel shows, rather than limiting the vertical extent of that panel.

Not right or wrong -- just saying we understand the consequence.

Reply:
Not sure that understand that, bottom toolbar docking area is a sibbling of the container window, isn't it?

Reply:
You are right, I misread that